testcontainers/src/clients/cli.rs

```diff
@@ -418,7 +421,7 @@ impl Docker for Cli {
                     self.stderr_logs(id).wait_for_message(&message).unwrap()
                 }
                 WaitFor::Duration { length } => {
-                    std::thread::sleep(length);
+                    sleep(length);
```

Comment:
Let's keep the diff minimal please.

Reply:
AFAIR it was highlighted by clippy locally - because 'extra' export, but sure, not related to this PR